core/ui/EditorToolbar/picture-dropdown.tid

```diff
@@ -4,8 +4,44 @@ title: $:/core/ui/EditorToolbar/picture-dropdown
 [img[$(imageTitle)$]]
 \end
 
+\define insert-external-image()
```

Comment:
OK, tested on my desktop browser, works like a charm, seems like a much needed improvement thanks for all the improvements.
So what's the plan to get this into official release? I no longer feel like this should be in my PR, it is after all your code skill here, not mine. :)